sass/base/components/footer/footer.twig

```diff
@@ -10,7 +10,7 @@
 
 
 <div class="footer__copyright">
-  <p>© Comic Relief 2017. Comic Relief, registered charity 326568 (England/Wales); SC039730 (Scotland)</p>
+  <p>© Comic Relief 2018. Comic Relief, registered charity 326568 (England/Wales); SC039730 (Scotland)</p>
```

Comment:
Agree withj Andy, We should definitely automate this,
```
{{ "now"|date("Y") }}
```

Reply:
Oh yeah..

Reply:
haha i **was** joking, but _nice!_